ocrd_detectron2/segment.py

```diff
@@ -187,45 +187,47 @@ class id to a new PAGE region type (and subtype).
             else:
                 zoomed = 1.0
 
-            # for morphological post-processing, we will need the binarized image, too
-            page_image_bin, _, _ = self.workspace.image_from_page(
-                page, page_id,
-                feature_selector='binarized')
-            # workaround for OCR-D/core#687:
-            if 0 < abs(page_image_raw.width - page_image_bin.width) <= 2:
-                diff = page_image_raw.width - page_image_bin.width
-                if diff > 0:
-                    page_image_raw = crop_image(
-                        page_image_raw,
-                        (int(np.floor(diff / 2)), 0,
-                         page_image_raw.width - int(np.ceil(diff / 2)),
-                         page_image_raw.height))
-                else:
-                    page_image_bin = crop_image(
-                        page_image_bin,
-                        (int(np.floor(-diff / 2)), 0,
-                         page_image_bin.width - int(np.ceil(-diff / 2)),
-                         page_image_bin.height))
-            if 0 < abs(page_image_raw.height - page_image_bin.height) <= 2:
-                diff = page_image_raw.height - page_image_bin.height
-                if diff > 0:
-                    page_image_raw = crop_image(
-                        page_image_raw,
-                        (0, int(np.floor(diff / 2)),
-                         page_image_raw.width,
-                         page_image_raw.height - int(np.ceil(diff / 2))))
-                else:
-                    page_image_bin = crop_image(
-                        page_image_bin,
-                        (0, int(np.floor(-diff / 2)),
-                         page_image_bin.width,
-                         page_image_bin.height - int(np.ceil(-diff / 2))))
+            # check wether input image is binarized
+            if page_image_info.photometricInterpretation == "1":
```

Comment:
That's a strange thing to query: it is only defined if the image was a TIFF (Pillow's TIFF plugin), and it does not even discern binarized vs others: binarized would be `.mode == '1'`. This [signifies `BlackIsZero`](https://github.com/python-pillow/Pillow/blob/5d070222d21138d2ead002fd33fdf5adcb708941/src/PIL/TiffTags.py#L105), which [could be true](https://github.com/python-pillow/Pillow/blob/5d070222d21138d2ead002fd33fdf5adcb708941/src/PIL/TiffImagePlugin.py#L1535-L1557) for 
- binarized
- grayscale 8-bit
- grayscale 8-bit plus alpha 8-bit
- grayscale 16-bit int
- grayscale 32-bit int
- grayscale 32-bit float

I don't understand the purpose yet: what did go wrong before?